### DISCUSSION TITLE
Possible to ignore certain queries in graphcache?

### DISCUSSION THREAD

**Discussion by GollyJer** [urql-graphql/urql#2292] [Q&A] [answered]
We're using React Native and running into an issue with the SQLite database getting full when using graphecache.

The easiest solution would be to tell the cache to completely ignore one of our queries that pulls in a ton of data but doesn't need to be cached and continue to cache the "smaller" queries.

Is this possible?
Thanks.

**Comment by JoviDeCroock** [comment, marked as answer]
You could try altering the storage-layer to filter a few entities out but apart from that currently no functionality, in the storage-write-data you could write something that bails out of writing

**Reply by GollyJer** [reply]
Sounds good. Thanks @JoviDeCroock
